bitcoin/src/psbt/mod.rs

```diff
@@ -59,6 +59,48 @@ pub struct Psbt {
     pub outputs: Vec<Output>,
 }
 
+#[derive(Debug, Clone, PartialEq)]
+/// This error is returned when extracting a Transaction from a Psbt.
+pub enum ExtractTxError {
```

Comment:
I'm getting really picky now, if I'm annoying you just ignore me :) 

I'd prefer to see this error code go down with the rest of the error code at the bottom of the file. In the past error code has been put at the top but over the last 18 months we've (well only me really) been moving it down the bottom (or into other files). This is based on the principle that its easier to read a file if the important stuff is at the top, error code is not important so it just clutters the place up if its at the top.

(I think because rust-bitcoin was written originally by @apoelstra and he came from C there was the habit of putting stuff before its used but we don't have to do this in Rust. I guessed this because I came from C as well and got taught this by a non-C dev :)